Makefile

```diff
@@ -72,7 +80,8 @@ serve: html
 
 
 clean: clean-doc
-	@test ! -e "$(VENVDIR)" -o -d "$(VENVDIR)" -a "$(abspath $(VENVDIR))" != "$(VENVDIR)"
+	@test ! -e "$(VENVDIR)" -o -d "$(VENVDIR)" \
+        -a "$(abspath $(VENVDIR))" != "$(VENVDIR)"
 	rm -rf $(VENVDIR)
```

Comment:
If `"$(VENVDIR)"` doesn't exist, `! -e "$(VENVDIR)"` will be TRUE. Short-circuit evaluation will yield TRUE for the whole @test statement, so make will continue instead of quitting: `rm -rf $(VENVDIR)` will be performed on a non-existent $VENVDIR.

Otherwise `make clean`'s working because:
- VENVDIR's set to `.sphinx/venv` which generally exists when `make clean` is run
- Therefore `! -e "$(VENVDIR)"` is FALSE, but there's a -o expression which @test therefore must evaluate
- `-d "$(VENVDIR"` is TRUE (`.sphinx/venv` is a directory), but there's a -a expression which must also be evaluated
- `"$(abspath $(VENVDIR))"` generally _isn't_ identical to `"$(VENVDIR)"`, so the -a expression is TRUE.
- FALSE OR TRUE AND TRUE yields TRUE, so make continues: `rm -rf $(VENVDIR)` is performed.

Where expressions are tied by operator associativity, it's good practice to wrap lines in a way that emphasises it rather than disguising it - it makes errors easier to spot. At present:

```
@test ! -e "$(VENVDIR)" -o -d "$(VENVDIR)" \
-a ...
```

keeps two expressions together with -o between them and splits the line between two expressions associated by -a, as if -o associates more tightly than -a. Ignoring problems with the logic, a representation less prone to being misinterpreted would be:

```
@test ! -e "$(VENVDIR)" -o \
-d "$(VENVDIR)" -a "$(abspath $(VENVDIR))" != "$(VENVDIR)"
```